### PR TITLE
Dont show message if emoji is blacklisted

### LINKED ISSUE #50: Not showing the quote message when resolving a request as ✅
It's a bit confusing and annoying that the quote block will show even if you resolve a request as ✅.

### PR DESCRIPTION
**Pull request by urielsalis**
Fixes #50

### CODE REVIEW

File: src/events/requests/ResolveRequestEventHandler.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ export default class ResolveRequestEventHandler implements EventHandler {
 
 		TaskScheduler.clearMessageTasks( reaction.message );
 
-		if ( BotConfig.request.prepend_response_message == PrependResponseMessageType.WhenResolved ) {
+		if ( BotConfig.request.prepend_response_message == PrependResponseMessageType.WhenResolved
+		   && BotConfig.request.ignore_prepend_response_message_emoji !== reaction.emoji.name ) {
```
Comment (SPGoding) [inline src/events/requests/ResolveRequestEventHandler.ts:21]: (Minor) I personally pefer putting `&&` at the end of the last line

Comment (NeunEinser) [reply src/events/requests/ResolveRequestEventHandler.ts:21]: I prefer it this way, and I think there are other places in the code where it's at the beginning of one line.

Comment (violine1101) [reply src/events/requests/ResolveRequestEventHandler.ts:21]: I prefer it in the front as well, makes it clearer what the condition is imo

**Comment by violine1101** [comment]
> Someone with access to the bot should merge this so the config can be adjusted.

Yeah, gotta work on #41 so that that is no longer necessary

**Pull request merged**